release-notes/25.03/index.html

```diff
@@ -0,0 +1,150 @@
+                </a>
+                <div class="dropdown-menu dropdown-menu-end">
+                  <a class="dropdown-item" href="https://www.cl.cam.ac.uk/research/security/ctsrd/cheri/cheri-slack.html"><i class="bi bi-slack me-2"></i>Talk to Us on Slack</a>
+                  <a class="dropdown-item" href="https://lists.cam.ac.uk/sympa/info/cl-cheribsd-announce"><i class="bi bi-megaphone me-2"></i>Sign Up for Email Updates</a>
+                  <a class="dropdown-item" href="https://lists.cam.ac.uk/sympa/info/cl-cheribsd-discuss"><i class="bi bi-chat-right-text me-2"></i>Join Our Email Discussion List</a>
+                </div>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </header>
+
+    <main class="flex-grow-1">
+      <section class="container py-3">
+        <div class="bg-light border rounded-3 p-4">
+          <h1>CheriBSD 25.03 Release Notes</h1>
+          <h2>Overview</h2>
+          <p>
+          Welcome to the Spring 2025 (25.03) release of the CheriBSD operating system.  As well as a general update of the baseline FreeBSD OS (corresponding to CURRENT as of January 2024) from which CheriBSD is derived, we have introduced several new research components which will be of interest to the CHERI and CheriBSD community:
+          </p>
+          <ul>
+            <li>
+            Library Compartmentalization (<a href="https://man.cheribsd.org/cgi-bin/man.cgi/release-25.03/c18n">https://man.cheribsd.org/cgi-bin/man.cgi/release-25.03/c18n</a>) has matured further and we are generally able to run with it enabled system wide.  It can be enabled system wide by setting the <code>security.cheri.lib_based_c18n_default</code> sysctl to <code>1</code>.  A new <code>procstat</code> command <code>compartments</code> allows instantiated compartments to be displayed.  While this implementation is now capable of running the full desktop, it is neither complete nor intended to be secure.
+            </li>
+            <li>
+	    Sub-library Compartmentalization has been added.  When compiled with teh <code>llvm-morello-c18n</code> compiler package and the <code>WITH_COMPARTMENTALIZTION_POLICY</code> make option set, libc will be compiled with a series of compartments.  Of particular interest, malloc and the malloc revocation wrapperm (mrs) are placed in their own compartments, protecting their powerful capabilities to the entire heap from the rest of libc.  Future releases will expand the use of such engineered compartments.
+            </li>
+            <li>
+	    To strengthen library compartmentalization, we have altered the function pointer ABI, introducing new relocations to allow function pointers to be wrapped by trampolines.  This functionality is currently disabled by default to easy source upgrades. It can be enabled system wide by setting the <code>security.cheri.lib_based_c18n_wrap_fptr</code> sysctl to <code>1</code>.  It will be the default in the next release.
```

Comment:
```suggestion
	    To strengthen library compartmentalization, we have altered the function pointer ABI, introducing new relocations to allow function pointers to be wrapped by trampolines.  This wrapping is currently disabled by default to ease upgrades. It can be enabled system wide by setting the <code>security.cheri.lib_based_c18n_wrap_fptr</code> sysctl to <code>1</code>.  It will be the default in the next release.
```

Reply:
Maybe "It will be enabled by default in the next release." for the last sentence?